src/web/lib/getBrazeUuid.ts

```diff
@@ -1,12 +1,23 @@
 import { joinUrl } from '@root/src/lib/joinUrl';
 import { callApi } from '@root/src/web/lib/api';
 
-export const getBrazeUuid = async (ajaxUrl: string): Promise<string> => {
+export const getBrazeUuid = async (ajaxUrl: string): Promise<string | void> => {
 	const url = joinUrl([ajaxUrl, 'user/me']);
 	return callApi(url, {
 		credentials: 'include',
 	})
-		.then((json) => json?.user?.privateFields?.brazeUuid)
+		.then(
+			(json: {
+				user?: {
+					privateFields?: {
+						brazeUuid?: string;
+					};
+				};
+			}) =>
+				json.user &&
+				json.user.privateFields &&
+				json.user.privateFields.brazeUuid,
```

Comment:
Can we still use `?.` for indexing here?

Reply:
yes I forgot we can since the node update thx 👍